### ISSUE TITLE
Configure Service Accounts for Pods page doesn't talk about the new BoundServiceAccountTokenVolume feature

### ISSUE THREAD

**Issue by fasaxc** [kubernetes/website#32655]
**This is a Bug Report**

**Problem:**
Configure Service Accounts for Pods page doesn't talk about the new BoundServiceAccountTokenVolume feature, it describes time-based expiration but tokens can now expire for other reasons.

**Proposed Solution:**
Add note about BoundServiceAccountTokenVolume and that it means that service account tokens are now revoked on pod termination.

**Page to Update:**
https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/

v1.21+

**Comment by liggitt** [comment]
> The service account token will also become invalid against the API when the Pod or the ServiceAccount is deleted.

**Comment by sftim** [comment]
I agree, the docs _do_ mention ServiceAccount token volume projection.

It would be nice to have a concept page about ServiceAccounts (see https://github.com/kubernetes/website/issues/31847); the task pages could then refer people to that concept page for further reading.

**Comment by tengqm** [comment]
/reopen
This is not supposed to be closed yet.

**Comment by sftim** [comment]
What's the remaining work on this issue?

**Comment by sftim** [comment]
https://kubernetes.io/docs/concepts/security/service-accounts/ is all about ServiceAccounts and frames the integration with Secret very much as a legacy thing.

For the specifics of this issue, I think https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/#manually-create-an-api-token-for-a-serviceaccount does cover this detail.

Are we happy to close this? I'm open to being persuaded there's more work to do.
